nerfstudio/field_components/mlp.py

```diff
@@ -15,13 +15,39 @@
 """
 Multi Layer Perceptron
 """
-from typing import Optional, Set, Tuple
+from typing import Literal, Optional, Set, Tuple, Union
 
 import torch
 from jaxtyping import Float
 from torch import Tensor, nn
 
 from nerfstudio.field_components.base_field_component import FieldComponent
+from nerfstudio.utils.printing import print_tcnn_speed_warning
+
+try:
+    import tinycudann as tcnn
+
+    TCNN_EXISTS = True
+except ImportError:
+    TCNN_EXISTS = False
+
+
+def activation_to_tcnn_string(activation: Union[nn.Module, None]) -> str:
+    """Converts a torch.nn activation function to a string that can be used to
+    initialize a TCNN activation function.
+
+    Args:
+        activation: torch.nn activation function
+    Returns:
+        str: TCNN activation function string
+    """
+    if isinstance(activation, nn.ReLU):
+        return "ReLU"
+    if isinstance(activation, nn.Sigmoid):
+        return "Sigmoid"
+    if isinstance(activation, type(None)):
+        return "None"
```

Comment:
Might as well add the others - https://github.com/NVlabs/tiny-cuda-nn/blob/master/DOCUMENTATION.md#activation-functions

Reply:
I added the ones for the nn.Module's and improved the error message.